terraform/projects/app-ecs-services/task-definitions/prometheus-server.json

```diff
@@ -18,7 +18,7 @@
       },
       {
         "sourceVolume": "alert-config",
-        "containerPath": "/etc/prometheus/alerts/alerts.default"
+        "containerPath": "/etc/prometheus/alerts"
```

Comment:
Interested in why this alerts config is a directory as opposed to the prometheus config which is a file?

Reply:
gut feel more than anything else.  We know we're going to manage other files within /etc/prometheus elsewhere (for the paas service discovery stuff) but I kind of feel we own all things in /etc/prometheus/alerts within terraform.  I could be wrong though.